### PR TITLE
Velocity estimates

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

**Comment by hamishwillee** [comment]
@peterbarker This failed with the following when trying to build the branch in vagrant:
```
g++ -I./util -I./ini -I./ini/cpp -Ijsoncpp -I.   -std=c++11 -g -DGIT_VERSION=\"v0.5-11-g7a2f\" -static -Wall -Werror -Wextra -Wunused -Wredundant-decls -D_FORTIFY_SOURCE=2 -Wfloat-equal -fsta
r=pointer-arith -Wpedantic -Wlogical-op     -c -o LA_MsgHandler.o LA_MsgHandler.cpp
LA_MsgHandler.cpp: In member function 'virtual void LA_MsgHandler_GPS::xprocess(const uint8_t*)':
LA_MsgHandler.cpp:316:14: error: redeclaration of 'uint32_t Spd'
     uint32_t Spd = require_field_int32_t(msg, "Spd")/(double)100.0f;
              ^
LA_MsgHandler.cpp:312:14: note: 'uint32_t Spd' previously declared here
     uint32_t Spd = require_field_int32_t(msg, "Spd")/(double)100.0f;
              ^
<builtin>: recipe for target 'LA_MsgHandler.o' failed
make: *** [LA_MsgHandler.o] Error 1
```

**Comment by hamishwillee** [comment]
@peterbarker Ignore that. I did a hard reset, clean, rebuild and it seems to work now.

File: analyzer/analyzer_velocity_estimate_divergence.cpp [hunk 1]
```diff
@@ -4,122 +4,119 @@
...
-//             _result[name]->set_delta_threshold(delta_warn());
-//             _result[name]->set_evilness(10);
-//         }
-//     }
-// }
+void Analyzer_Velocity_Estimate_Divergence::open_result(const std::string name,
+                                                        double delta)
+{
+    _result[name] = new Analyzer_Velocity_Estimate_Divergence_Result(name);
+   _result[name]->set_reason("This velocity estimate differs from the canonical craft velocity");
```
Comment (hamishwillee) [inline analyzer/analyzer_velocity_estimate_divergence.cpp]: indentation.

**Comment by peterbarker** [comment]
@hamishwillee Indentation will be fixed by the next patch, which factors a whole bunch of code.

**Pull request merged**